src/PostgREST/AppState.hs

```diff
@@ -96,15 +107,24 @@ initWithPool pool conf = do
     <*> newIORef 0
     <*> pure (pure ())
 
-  deb <-
+
+  debLogTimeout <-
     let oneSecond = 1000000 in
     mkDebounce defaultDebounceSettings
        { debounceAction = logPgrstError appState SQL.AcquisitionTimeoutUsageError
        , debounceFreq = 5*oneSecond
        , debounceEdge = leadingEdge -- logs at the start and the end
        }
 
-  return appState { debounceLogAcquisitionTimeout = deb }
+  debWorker <-
+    let decisecond = 100000 in
```

Comment:
This is enough to ensure schema cache reloading is still fast(io tests pass) but we still debounce.

Also tried manually with a bash loop `for ((;;)) { kill -s SIGUSR1 $(pgrep -n postgrest); }` and it's working fine.